google/cloud/storage/testing/storage_integration_test.cc

```diff
@@ -121,19 +121,13 @@ std::unique_ptr<RetryPolicy> StorageIntegrationTest::TestRetryPolicy() {
       .clone();
 }
 
-std::string StorageIntegrationTest::MakeRandomBucketName(std::string prefix) {
+std::string StorageIntegrationTest::MakeRandomBucketName() {
   // The total length of this bucket name must be <= 63 characters,
-  char constexpr kPrefix[] = "gcs-cpp-test-bucket-";  // NOLINT
+  char constexpr kPrefix[] = "cloud-cpp-testing-";  // NOLINT
   auto constexpr kMaxBucketNameLength = 63;
   static_assert(kMaxBucketNameLength > sizeof(kPrefix),
                 "The bucket prefix is too long");
```

Comment:
It looks like this constant/check no longer applies here.  Indeed, `testing::MakeRandomBucketName()` adds to the prefix, so the check should really happen there (or at least it shouldn't wrap `max_random_characters`).

Reply:
Good point, removed.